pkg/model/awsmodel/autoscalinggroup.go

```diff
@@ -60,18 +60,17 @@ func (b *AutoscalingGroupModelBuilder) Build(c *fi.ModelBuilderContext) error {
 					return err
 				}
 			}
-			volumeType := fi.StringValue(ig.Spec.RootVolumeType)
-			volumeIops := fi.Int32Value(ig.Spec.RootVolumeIops)
 
-			switch volumeType {
-			case "io1":
-				if volumeIops == 0 {
-					volumeIops = DefaultVolumeIops
-				}
-			default:
+			volumeType := fi.StringValue(ig.Spec.RootVolumeType)
+			if volumeType == "" {
 				volumeType = DefaultVolumeType
 			}
 
+			volumeIops := fi.Int32Value(ig.Spec.RootVolumeIops)
+			if volumeIops == 0 {
+				volumeIops = DefaultVolumeIops
```

Comment:
Is the `DefaultVolumeIOps` okay for the other Volume types? I imagine it's fine between gp2 and io1 but should this be different for standard?

Reply:
The actual `RootVolumeIops` of the launch configuration is set only if the `volumeType` is `io1`. Please see line 90.

https://github.com/int128/kops/blob/d97e0e6205ec9f1d0d79d84afc943cf8774ab71d/pkg/model/awsmodel/autoscalinggroup.go#L90-L92